src/cf-layout/src/cf-layout.less

```diff
@@ -650,36 +650,35 @@
     });
   }
 }
-//
-// Hero
-//
+
 
 @hero-desktop-height: 285px;
 @hero-image-height: 195px;
-@desktop-min: 801px;
+@hero-bg: #f7f8f9;
+@hero-overlay-bg: #5a5d61;
+@hero-overlay-text: white;
 
 .hero {
-    background-color: @block__bg;
+    background-color: @hero-bg;
 
-    &_wrapper {
-        box-sizing: border-box;
-        padding: unit(@grid_gutter-width / @base-font-size-px, em)
-                 unit(15px / @base-font-size-px, em);
+    .respond-to-min( @bp-sm-min, {
+        display: table;
```

Comment:
From a quick test on tax-time-saving, moving `display: table;` to `.hero` breaks the `vertical-align: middle;` on `.hero_text` and `.hero_image`.